designs/developer/add-45/sections/pipelines.md

```diff
@@ -0,0 +1,80 @@
+---
+parent: Developer
+version: 4.5
+---
+
+# Pipelines section in Add Flows (added to 4.3)
+The Pipelines section allows the user to enable a pipeline template for their application, component or service.
+* A Pipelines section appears in +Add flows (for Import from Git,  Developer Catalog, and From Dockerfile) when the Pipelines operator is installed.
+* When the Pipelines operator is not installed, this section is not displayed
+* The Pipelines section has a “Dev Preview” badge
+
+## Import from Git
+### No Builder Image Selected
+* When a builder image has not been selected, the Pipelines option shall display an Info alert that says “Select a builder image to see if there is a pipeline template available for this runtime.”
+
+![Add pipeline](img/+Add_Flow_Pipelines_ImportFromGit_No_Img.png)
+
+### Builder Image Selected and Pipeline Template Exists
+* When a builder image has been selected, if a pipeline template is available, the Pipeline section shall be updated to display an “Add pipeline”  checkbox option
+* The default state of the Add pipeline checkbox is OFF
+* An expandable control with text “Show pipeline visualization” is displayed below the Add pipeline checkbox and is collapsed by default
+
+![Add pipeline](img/+Add_Flow_Pipelines_ImportFromGit.png)
```

Comment:
if you keep the relative path here, you'll have to add the image in the designs/developer/add-45/sections/img directory, or you can point back to the 4.4 image.

Reply:
@serenamarie125 OK